lib/node_modules/@stdlib/math/base/special/gcdf/README.md

```diff
@@ -0,0 +1,243 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2018 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# gcdf
+
+> Compute the [greatest common divisor][gcd] (gcd).
```

Comment:
The description here and everywhere else should indicate that this package is for single-precision numbers. We can have something like:

```
Compute the [greatest common divisor][gcd] (gcd) of two single-precision floating point numbers.
```